src/task/task.controller.ts

```diff
@@ -12,27 +14,39 @@ export class TaskController {
 	) { }
 
 	@Post()
-	public addTask(@Body() taskDTO: TaskDTO): Promise<InsertResult> {
-		return this.taskService.addTask(taskDTO);
+	@UseGuards(JwtAuthGuard)
```

Comment:
try to add this guard under controller decorator
@Controller("task")
@UseGuards(JwtAuthGuard)